graylog2-web-interface/src/components/indices/IndexSetFieldTypeProfiles/EditProfile.tsx

```diff
@@ -59,8 +63,10 @@ const EditProfile = ({
     navigate(Routes.SYSTEM.INDICES.FIELD_TYPE_PROFILES.OVERVIEW);
   }, [navigate, sendTelemetry, telemetryPathName]);
 
+  const initialValues = useMemo(() => omit(profile, ['id', 'indexSetIds']), [profile]);
```

Comment:
I think instead of using `omit` we should provide only the values we need in the form or new attributes will end up in the form state and could cause an error on submit.